apps/flipper/pubspec.yaml

```
@@ -144,70 +118,62 @@ dependencies:
-  amplify_core: ^2.6.1
-  amplify_api: ^2.6.1
+  sqlite3_flutter_libs: ^0.5.39
+  amplify_core: ^2.6.5
+  amplify_api: ^2.6.5
   amplify_flutter: ^2.6.1
-  amplify_auth_cognito: ^2.6.1
-  amplify_storage_s3: ^2.6.1
+  amplify_auth_cognito: ^2.6.5
+  amplify_storage_s3: ^2.6.5
   shared_preferences: ^2.5.2
   easy_localization: ^3.0.7 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)